src/main/webapp/populateData.js

```diff
@@ -70,16 +70,22 @@ function populateTasks() {
         }
         return response.json();
       })
-      .then((tasksList) => {
-        // Convert JSON to string containing all task titles
-        // and display it on client
-        if (tasksList.length !== 0) {
-          const tasks =
-              tasksList.map((a) => a.title).reduce((a, b) => a + '\n' + b);
-          tasksContainer.innerText = tasks;
-        } else {
-          tasksContainer.innerText = 'No tasks found';
-        }
+      .then((tasksResponse) => {
+        document
```

Comment:
This is a really clean way of using the query selectors! I'm going to use this in my future PRs (I don't know why but I didn't think to chain them like this. Thanks!)

Reply:
Thanks! Honestly, it was to shush the JavaScript linter.

Reply:
`RESOLVED`